index.html

```diff
@@ -1089,12 +1089,23 @@ <h2>
               </li>
             </ol>
           </li>
-          <li>Present a user interface that will allow the user to interact
-          with the |handlers|. The user agent SHOULD prioritize the preference
-          of the user when presenting payment methods. It is RECOMMENDED that
-          the language of the user interface match the [=node/language=] of the
-          <a data-cite="HTML/dom.html#the-body-element-2">the body element</a>
-          element.
+          <li>
+            <p>
+              Present a user interface that will allow the user to interact
+              with the |handlers|. The user agent SHOULD prioritize the
+              preference of the user when presenting payment methods.
```

Comment:
```suggestion
              user's preference when presenting payment methods.
```
* clearer and shorter wording 
* also - it is not very clear to me what `preferences` users are allowed to have. could you clarify this?